command_test.go

```diff
@@ -24,13 +24,22 @@ func TestCommand_String(t *testing.T) {
 		t.Errorf("expected %q to be %q", got, want)
 	}
 
-	cmd.Message = "bar"
+	cmd = Command{Name: "foo", Message: "bar"}
 	if got, want := cmd.String(), "::foo::bar"; got != want {
 		t.Errorf("expected %q to be %q", got, want)
 	}
 
-	cmd.Properties = CommandProperties{"bar": "foo"}
+	cmd = Command{
+		Name:       "foo",
+		Message:    "bar",
+		Properties: CommandProperties{"bar": "foo"},
+	}
```

Comment:
I put it in the commit message, but this change (and the one on line 27) is just a nit by me. I figured "new `Command{}` literal" is slightly better than "modify existing `Command` object" because it visually "localizes" the value being tested.

Happy to revert my editorializing though. (The real goal of the edits was to test the unset `Name` on line 41.)